siriuspy/siriuspy/devices/idff.py

```diff
@@ -31,9 +26,9 @@ def __init__(self, devname):
         self._devname = devname  # needed for _create_devices
         self._idffconfig = _IDFFConfig()
 
+        self._pol_mon = _ID.get_idclass(devname).PARAM_PVS.POL_MON
```

Comment:
I think this line could be after defining `self._devid` and we could use `self._devid.PARAM_PVS.POL_MON` instead of `ID.get_idclass`

Reply:
you see that I need `pol_mon` in `props2init` before I can create the ID device? But I indeed can simplify things after I changed how I am managing `None` value for `POL_MON` in method `_create_devices`...

Reply:
done